[CI:DOCS] Document volume mounts of source directories do NOT get created (#8519)
containers/podman: docs/source/markdown/podman-create.1.md

#### **--replace**=**true**|**false**

> If another container with the same name already exist, replace and remove it. The default is **false**.

Reviewer: ```suggestion
If another container with the same name already exists, replace and remove it. The default is **false**.
```

Reviewer: Sorry, I screwed this one up earlier, "exists" is correct for third person singular.  "exist" is good for you, we, they, or you.

Create a bind mount. If you specify, ` -v /HOST-DIR:/CONTAINER-DIR`, Podman
bind mounts `/HOST-DIR` in the host to `/CONTAINER-DIR` in the Podman
container. Similarly, `-v SOURCE-VOLUME:/CONTAINER-DIR` will mount the volume

> in the host to the container. If no such named volume exist, Podman will

Reviewer: ```suggestion
in the host to the container. If no such named volume exists, Podman will
```

The `CONTAINER-DIR` must be an absolute path such as `/src/docs`. The volume
will be mounted into the container at this directory.

Volumes may specify a source as well, as either a directory on the host
or the name of a named volume. If no source is given, the volume will be created as an

> anonymous named volume with a randomly generated name, and will be removed when

Reviewer: old, but could you change "anonymous" to "anonymously"

the container is removed via the `--rm` flag or `podman rm --volumes`.

If a volume source is specified, it must be a path on the host or the name of a
named volume. Host paths are allowed to be absolute or relative; relative paths
are resolved relative to the directory Podman is run in. If the source does not

> exists, Podman will return and error. Users must pre-create the source files or

Reviewer: ```suggestion
exist, Podman will return an error. Users must pre-create the source files or
```

Author: fixed

directories.

Any source that does not begin with a `.` or `/` will be treated as the name of
a named volume. If a volume with that name does not exist, it will be created.

> Volumes created with names are not anonymous. They are not removed by the `--rm`

Reviewer: suggest just to be clear that you're talking about named and not anonymous volumes
```suggestion
Volumes created with names are not anonymous and they are not removed by the `--rm`
```

Author: fixed
